src/components/icons/connector_icons.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import azure_blob_storage from '../../../public/assets/source_icons/azure_blob_storage.svg';
+import box from '../../../public/assets/source_icons/box.svg';
+import confluence_cloud from '../../../public/assets/source_icons/confluence_cloud.svg';
+import custom from '../../../public/assets/source_icons/custom.svg';
+import dropbox from '../../../public/assets/source_icons/dropbox.svg';
+import github from '../../../public/assets/source_icons/github.svg';
+import gmail from '../../../public/assets/source_icons/gmail.svg';
+import google_cloud_storage from '../../../public/assets/source_icons/google_cloud_storage.svg';
+import google_drive from '../../../public/assets/source_icons/google_drive.svg';
+import jira_cloud from '../../../public/assets/source_icons/jira_cloud.svg';
+import mongodb from '../../../public/assets/source_icons/mongodb.svg';
+import microsoft_sql from '../../../public/assets/source_icons/mssql.svg';
+import mysql from '../../../public/assets/source_icons/mysql.svg';
+import network_drive from '../../../public/assets/source_icons/network_drive.svg';
+import onedrive from '../../../public/assets/source_icons/onedrive.svg';
+import oracle from '../../../public/assets/source_icons/oracle.svg';
+import outlook from '../../../public/assets/source_icons/outlook.svg';
+import postgresql from '../../../public/assets/source_icons/postgresql.svg';
+import amazon_s3 from '../../../public/assets/source_icons/s3.svg';
+import salesforce from '../../../public/assets/source_icons/salesforce.svg';
+import servicenow from '../../../public/assets/source_icons/servicenow.svg';
+import sharepoint from '../../../public/assets/source_icons/sharepoint.svg';
+import sharepoint_online from '../../../public/assets/source_icons/sharepoint_online.svg';
+import slack from '../../../public/assets/source_icons/slack.svg';
+import teams from '../../../public/assets/source_icons/teams.svg';
+import zoom from '../../../public/assets/source_icons/zoom.svg';
```

Comment:
Wonder if we can also get these so they're not relative paths, but aliased?

Is this more difficult because we're using EuiIcon? Would it be easier to use Next's `<Image />` component?